src/content/docs/tutorial-reporting-data/capture-web-telemetry.mdx

```diff
@@ -0,0 +1,354 @@
+    </Step>
+    <Step>
+## Check AJAX call grouping [#ajax-grouping]
+After you check your page views, you should do the same for your AJAX call grouping. AJAX calls are grouped to make it is easier to navigate them at scale. Sometimes there are so many AJAX calls that navigating them by individual request URL becomes difficult. Use the UI or a NRQL query to check if you need to adjust AJAX grouping. 
+
+<CollapserGroup>
+  <Collapser
+    id="js-stack-trace-ui"
+    title="Via the UI"
+  >
+  Check AJAX grouping for your app by selecting it from the left hand menu and grouping by '_groupedRequestUrl_'. If what you see looks a lot the screen shot below, make a note and follow the instructions in this guide on how to address it. 
+ <img src={omaOeScBrowserGroupedAjaxRequests} alt="AJAX Grouping" title="AJAX Grouping" />  
+
+  </Collapser>
+  <Collapser
+    id="js-stack-trace-nrql"
+    title="Via NRQL"
+  >
+  Run the following query: 
+```
+SELECT count(*) FROM JavaScriptError WHERE appName = _your app name_ AND stackTrace IS NOT NULL AND stackTrace NOT LIKE '' SINCE 1 WEEK AGO 
+```
+  A 0 count means that no JavaScript errors have been captured. 
+
+  You can check all your web applications in a sub account by running the following: 
+```
+SELECT count(*) FROM JavaScriptError WHERE stackTrace IS NOT NULL AND stackTrace NOT LIKE '' FACET appName LIMIT MAX SINCE 1 WEEK AGO
+```
+Web applications not present in the results don't have JavaScript errors with stack traces. 
+  </Collapser>
+</CollapserGroup>
+
+Use [Segment allow lists](/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls/) to tune how your AJAX Requests are being grouped.         
+    </Step>
+    <Step>
+## Enable distributed tracing [#distributed-tracing]
+Next, enable [distributed tracing in Browser](/docs/browser/new-relic-browser/browser-pro-features/browser-data-distributed-tracing/) to help you improve AJAX performance by tracing requests to the backend all the way to the final endpoint. Tracing information is useful for understanding which applications are impacting user experience. You can use this information to address services issues yourself or delegate to the team responsible.         
+    </Step>
+    <Step>
+## Set up deployments [#deployments]
+Next, underlyingse NerdGraph to [track changes in your web application](/docs/change-tracking/change-tracking-graphql/) so you can see the impact of changes you make against performance KPIs, conversions, and user engagement.        
+    </Step>
+    <Step>
+## Add Custom attributes [#custom]
+Use [custom attributes](/docs/data-apis/custom-data/custom-events/report-custom-event-data/#ways) to filter and group data. Custom attributes are optional but provide a lot of value. Below are the most commonly recommended attributes. Many customers add more. 
+    </Step>
+<CollapserGroup>
+  <Collapser
+    id="user-attribute"
+    title="user"
```

Comment:
Should these all be wrapped in code?  ``user` `

Reply:
I'm not sure that's required here since everything is already framed as being an attribute?